config/migrate.ts

```diff
@@ -0,0 +1,209 @@
+import * as config_DEPRECATED from './config_DEPRECATED';
+import { CLIConfiguration } from './CLIConfiguration';
+import {
+  CLIConfig,
+  CLIConfig_DEPRECATED,
+  CLIConfig_NEW,
+  Environment,
+} from '../types/Config';
+import { CmsPublishMode } from '../types/Files';
+import {
+  writeConfig,
+  createEmptyConfigFile,
+  loadConfig,
+  deleteEmptyConfigFile,
+} from './index';
+import {
+  getConfigFilePath,
+  configFileExists as newConfigFileExists,
+  deleteConfigFile as newDeleteConfigFile,
+} from './configFile';
+import {
+  GLOBAL_CONFIG_PATH,
+  DEFAULT_CMS_PUBLISH_MODE,
+  HTTP_TIMEOUT,
+  ENV,
+  HTTP_USE_LOCALHOST,
+  DEFAULT_ACCOUNT,
+  DEFAULT_PORTAL,
+} from '../constants/config';
+import { i18n } from '../utils/lang';
+
+const i18nKey = 'config.migrate';
+
+export function getConfig(
+  useHiddenConfig?: boolean,
+  configPath?: string
+): Partial<CLIConfig> | null {
+  if (useHiddenConfig) {
+    return CLIConfiguration.config;
+  }
+  return config_DEPRECATED.loadConfig(configPath);
+}
+
+export function getConfigPath(
+  configPath?: string,
+  useHiddenConfig = false
+): string | null {
+  if (useHiddenConfig) {
+    return getConfigFilePath();
+  }
+  return config_DEPRECATED.getConfigPath(configPath);
+}
+
+export function configFileExists(
+  useHiddenConfig?: boolean,
+  configPath?: string
+): boolean {
+  return useHiddenConfig
+    ? newConfigFileExists()
+    : Boolean(config_DEPRECATED.getConfigPath(configPath));
+}
+
+export function deleteConfigFile(useHiddenConfig = false): void {
+  if (useHiddenConfig) {
```

Comment:
Will we ever want this to delete the global config file? I'm wondering if we can get rid of this util and just call `config_DEPRECATED.deleteConfigFile()` directly wherever we need it